chat.go

```diff
@@ -428,3 +428,78 @@ func (c *Client) UpdateUserChatColor(params *UpdateUserChatColorParams) (*Update
 
 	return update, nil
 }
+
+type SendChatMessageParams struct {
+	// The ID of the broadcaster whose chat room the message will be sent to
+	BroadcasterID string `query:"broadcaster_id"`
+
+	// The ID of the user sending the message. This ID must match the user ID in the user access token
+	SenderID string `query:"sender_id"`
+
+	// The message to send. The message is limited to a maximum of 500 characters.
+	// Chat messages can also include emoticons.
+	// To include emoticons, use the name of the emote.
+	// The names are case sensitive.
+	// Don’t include colons around the name (e.g., :bleedPurple:).
+	// If Twitch recognizes the name, Twitch converts the name to the emote before writing the chat message to the chat room
+	Message string `query:"message"`
+
+	// The ID of the chat message being replied to
+	ReplyParentMessageID string `query:"reply_parent_message_id"`
```

Comment:
Thats a POST request, so it should be body fields.

Also `reply_parent_message_id`, so there should be `omitempty` on this field.

Reply:
https://dev.twitch.tv/docs/api/reference/#send-chat-message

Reply:
Sorry for my inattention. Fixed